apps/web/src/studio/components/workflows/preferences/types.ts

```diff
@@ -10,5 +10,4 @@ export type SubscriptionPreferenceRow = {
 export type WorkflowGeneralSettings = {
   workflowId: string;
   name: string;
-  description?: string;
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Don't forget to remove the setting of description [here‏](https://github.com/novuhq/novu/blob/87960d31509318a41dc2dac569051836bc76c5d4/apps/web/src/pages/templates/editor_v2/useWorkflowDetailPageForm.ts#L66)